xrpl/utils/tx_parser.py

```diff
@@ -0,0 +1,197 @@
+from xrpl.utils.xrp_conversions import drops_to_xrp
+import pydash
+
+"""
+This is a rewritten repository of https://github.com/ripple/ripple-lib-extensions/blob/master/transactionparser/src/balancechanges.js in python.
+"""
+
+### Utils ###
+class Normalized_Node():
+    def __init__(self, difftype: str, entryType: str, ledgerIndex: str, newFields: dict, finalFields: dict, previousFields: dict) -> None:
+        self.DiffType = difftype
```

Comment:
```suggestion
        self.diff_type = difftype
```
Please use `snake_case` for variable names.